apollo-router-scaffold/templates/base/xtask/Cargo.toml

```diff
@@ -13,7 +13,7 @@ apollo-router-scaffold = { path ="{{integration_test}}apollo-router-scaffold" }
 {{#if branch}}
```

Comment:
This comment isn't about this line but rather about `/apollo-router-scaffold/Cargo.toml` and its `version` property: It's currently `0.11.0`, so similar thing to my above comment.